lib/atp_scraper/activities/record.rb

```diff
@@ -1,7 +1,17 @@
 module Activities
   # Activity Record Class
   class Record
-    def self.pickup_record(record_doc)
+    def initialize(doc)
+      @record = doc
+    end
+
+    def info
+      pickup_record(@record)
+    end
+
+    private
+
+    def pickup_record(record_doc)
```

Comment:
__[reek]__
* [UtilityFunction] doesn't depend on instance state (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Utility-Function.md)